pkg/capi/infrastructure/aws.go

```diff
@@ -83,7 +83,7 @@ func NewAWSDeployOptions() *AWSDeployOptions {
 		NodeMachineType:         "t3.large",
 		NodeIAMProfile:          "CAPI_AWS_Worker",
 		CloudProviderVersion:    "v1.20.0-alpha.0",
-		CalicoVersion:           "v3.18",
+		CalicoVersion:           "v3.24.1",
```

Comment:
should  we enable renovate to get updates here?

Reply:
I'm kind of thinking this should be re-implemented to allow overrides per-project (?)

Reply:
yeh that makes more sense

Reply:
i.e. should we bump or keep it stable? idk